waku/waku_store/common.nim

```diff
@@ -3,104 +3,115 @@ when (NimMajor, NimMinor) < (1, 4):
 else:
   {.push raises: [].}
 
-import std/[options, sequtils], stew/results, stew/byteutils, nimcrypto/sha2
+import std/[options], stew/results
 import ../waku_core, ../common/paging
 
 const
-  WakuStoreCodec* = "/vac/waku/store/2.0.0-beta4"
+  WakuStoreCodec* = "/vac/waku/store-query/3.0.0"
```

Comment:
I used the name from the spec. @jm-clius why?

Reply:
> Nitpick: why add the `-query` part?

Because that actually makes sense. :) 
The protocol is not used to Store a message in the node rather querying for data from a Store node.